src/encoded/types/shared_calculated_properties.py

```diff
@@ -284,7 +284,17 @@ class CalculatedVisualize:
     def visualize(self, request, hub, accession, assembly, status, files):
         hub_url = urljoin(request.resource_url(request.root), hub)
         viz = {}
-        for assembly_name in assembly:
+        vis_assembly = set()
+        viewable_file_formats = ['bigWig', 'bigBed']
+        viewable_file_status = ['released','in progress']
```

Comment:
This is a pep8 issue.  Need a space after commas.  
```
viewable_file_status = ['released', 'in progress']
```
Do you have linting running in your texteditor?  It's tough to use since there are so many pep8 issues preexisting in our files, but we can go over how it can be helpful.

Also, since we're now using this list in two different locations, we should add it at the top of the file as a constant.
```
VIEWABLE_FILE_STATUS = ['released','in progress']
```